copy-packages doesn't work

I have the following global packages installed under `v0.10.29`:
1. autoprefixer
2. bower
3. david
4. eslint
5. grunt-cli
6. grunth-cli
7. http-server
8. jshint
9. marked
10. node-gyp
11. npm
12. recursive-blame
13. uglify-js
14. yo

I've just installed Node 0.10.30 and this is what I get:

```
$ nvm --version
0.12.2
$ echo $NVM_DIR
/Users/mgol/.nvm
$ nvm ls
    v0.10.29
->  v0.10.30
    v0.11.13
      system
default -> 0.11 (-> v0.11.13)
$ nvm copy-packages v0.10.29
npm ERR! TypeError: Cannot read property 'latest' of undefined
npm ERR!     at next (/Users/mgol/.nvm/v0.10.30/lib/node_modules/npm/lib/cache/add-named.js:219:35)
npm ERR!     at /Users/mgol/.nvm/v0.10.30/lib/node_modules/npm/lib/cache/add-named.js:207:5
npm ERR!     at saved (/Users/mgol/.nvm/v0.10.30/lib/node_modules/npm/node_modules/npm-registry-client/lib/get.js:167:7)
npm ERR!     at Object.oncomplete (fs.js:107:15)
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Darwin 13.3.0
npm ERR! command "/Users/mgol/.nvm/v0.10.30/bin/node" "/Users/mgol/.nvm/v0.10.30/bin/npm" "install" "-g" "              "
npm ERR! cwd /Users/mgol
npm ERR! node -v v0.10.30
npm ERR! npm -v 1.4.21
npm ERR! type non_object_property_load
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/mgol/npm-debug.log
npm ERR! not ok code 0
```

--- a/nvm.sh
+++ b/nvm.sh
@@ -789,7 +789,7 @@ nvm() {
 
       # declare local INSTALLS first, otherwise it doesn't work in zsh
       local INSTALLS
-      INSTALLS=$(nvm use $VERSION > /dev/null && npm list --global --parseable --depth=0 2> /dev/null | tail -n +2 | \grep -o -e '/[^/]*$' | cut -c 2- | \grep -v npm | xargs)
+      INSTALLS=$(nvm use "$VERSION" > /dev/null && npm list -g --depth=0 | tail -n +2 | \grep -o -e ' [^@]*' | cut -c 2- | \grep -v npm | xargs)
 
       echo "$INSTALLS" | xargs npm install -g --quiet
     ;;

--- a/test/slow/nvm copy-packages/should work as expected
+++ b/test/slow/nvm copy-packages/should work as expected
@@ -11,10 +11,10 @@ EXPECTED_PACKAGES="autoprefixer bower david eslint grunt-cli grunth-cli http-ser
 echo "$EXPECTED_PACKAGES" | xargs npm install -g --quiet
 
 nvm use 0.10.29
-ORIGINAL_PACKAGES=$(npm list --global --parseable --depth=0 2> /dev/null | tail -n +2 | \grep -o -e '/[^/]*$' | cut -c 2- | \grep -v npm | sort | uniq | xargs)
+ORIGINAL_PACKAGES=$(npm list -g --depth=0 | tail -n +2 | \grep -o -e ' [^@]*' | cut -c 2- | \grep -v npm | sort | uniq | xargs)
 
 nvm copy-packages 0.10.28
-FINAL_PACKAGES=$(npm list --global --parseable --depth=0 2> /dev/null | tail -n +2 | \grep -o -e '/[^/]*$' | cut -c 2- | \grep -v npm | sort | uniq | xargs)
+FINAL_PACKAGES=$(npm list -g --depth=0 | tail -n +2 | \grep -o -e ' [^@]*' | cut -c 2- | \grep -v npm | sort | uniq | xargs)
 
 [ "$FINAL_PACKAGES" = "$EXPECTED_PACKAGES" ] || die "final packages ($FINAL_PACKAGES) did not match expected packages ($EXPECTED_PACKAGES)"
 [ "$ORIGINAL_PACKAGES" != "$FINAL_PACKAGES" ] || die "original packages matched final packages ($ORIGINAL_PACKAGES)"

